extensions/kafka/src/test/java/com/hazelcast/jet/kafka/impl/StreamKafkaPTest.java

```diff
@@ -169,11 +169,6 @@ public void when_projectionFunctionProvided_then_appliedToReadRecords() {
         });
     }
 
-    @Test
-    public void when_processingGuaranteeNone_then_ignorePartitionsInitialOffsets() {
```

Comment:
is it possible to test those scenarios with processing guarantee NONE? The scenarios were fine but it cannot be tested by only observing the output of the pipeline. Maybe we should just temporarily ignore the tests until we have better way to test that?

Reply:
I don't think so. We can test if there are any rows processed, but I don't think that's a good test

Reply:
Is it necessary to delete this particular test? I think only test scenarios that restart the job are affected by these random failures i.e. :

- `when_processingGuaranteeNone_then_continueFromBeginningAfterJobRestart`
- `when_processingGuaranteeNoneWithConsumerGroup_then_continueFromLastReadMessageAfterJobRestart`

(but I might be wrong)

Reply:
With processingGuarantee NONE you cannot guarantee any behaviour in regard to which items will be reprocessed and which not